### ISSUE TITLE
BurnSeverity Metrics

### ISSUE THREAD

**Issue by EarthScientist** [ua-snap/alfresco_postprocessing#3]
We need a class for BurnSeverity metrics, which (currently) is as simple as counts of the burn severity classes in a given subdomain in a given year.

**Comment by EarthScientist** [comment]
This is how we can wrangle this new data class once we have extracted it using `alfresco_postprocessing`:

```python
# working with / wrangling the new burnseverity data...  
# unsure yet where this will fit in overall, but wanted to get it on paper
import pandas as pd
import os

# this is the output from the alf_pp tool
pp = alfresco_postprocessing_output_TinyDB # [non-working example line]

# name of the domain we want to slice the data out of <<- we can only do one at a time 
# max and that is with long-form output data written to CSV...  One per domain. 
# other slicing can be performed but this is the easiest way so far.
domain_name = 'AOI_SERDP'
output_path = '/atlas_scratch/malindgren/TEST_BURNSEVERITY'

# pull out the burnseverity 
burnseverity = { (rec[ 'replicate' ],rec[ 'fire_year' ]):rec[ 'severity_counts' ][ domain_name ] for rec in pp.all() }

# slice the domain, this could be looped for multiples.
series_dict = { i:pd.Series( j ) for i,j in burnseverity.items() }

# put it all back together and fill in missing value places.
burnseverity_df = pd.concat( series_dict ).astype( int ).unstack().fillna( 0 )

# write to disk...
output_filename = os.path.join( output_path, 'burnseverity_counts_{}.csv'.format( domain_name ) )
burnseverity_df.to_csv( output_filename )

# read it back in like this, with a MultiIndex for Replicate Slicing:
bs_df = pd.read_csv( output_filename, index_col=[0,1], sep=',' )
```

**Comment by EarthScientist** [comment]
merged with #7 PR.  closing.
